int.ts

```diff
@@ -84,20 +84,10 @@ module J2ME {
       return reference;
     }
 
-    if (reference === null) {
-      return reference;
+    if (reference === null || reference === 0) {
+      return 0;
```

Comment:
Can we make null a constant? It's going to be much harder to find these changes after this pull request.